packages/core/src/api/clipboard/__snapshots__/childToParent.html

```diff
@@ -0,0 +1 @@
+<h2 data-text-color="red" data-level="2">Heading 1</h2><p>Nested Paragraph 1</p>
```

Comment:
I'm confused; shouldn't these snapshots have the prosemirror slice info?

Reply:
Ah wait, you test te externalhtml with snapshots, but the clipboard by calling paste, is that the idea?